src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+“You know,” Large said, “I always hated it when my fucking old man would rattle on and hit me with advice, but shit. Guess I’m older now than he was then.” He lit a cigarette and inhaled. “Always keep the family at the front of your mind.”
 
-“It breaks my heart dude, but it had to be said. Ninjas today just don’t know.”
+I tried to nod back, conveying a wisdom that I didn’t have yet, and we parted ways.
 
-“They don’t, man! The Dark Carnival saved my life, dude, and it’s about love. Shit is about family, dude. If these ninjas were out there listening to the shit...”
+[As I walked back towards the front of The Gutter:
 
-“Listen to the shit!!”
+>>[I dodged a hole in the ground.]<dodgeTrigger|
+>>[my dumb ass tripped over a hole in the ground.]<tripTrigger|]<tripDodgeChoice|
+\(click: ?dodgeTrigger)[(replace: ?tripDodgeChoice)[$dissolve[As I walked back towards the front of The Gutter, I smoothly sidestepped to dodge a hole in the ground.
 
-The shorter ninja started jumping and kicking an unknown assailant at this point. Large picked up on the energy took his hands away from his beard. He started punching one hand into the other to punctuate his points.
+[[Hell yeah|The Gutter - Outside hub]].]]]
+\(click: ?tripTrigger)[(replace: ?tripDodgeChoice)[$dissolve[As I walked back towards the front of The Gutter, my dumb ass tripped over a hole in the ground.
 
-“Exactly! If they were listening; like Dark Lotus? “Juggalo Family?” That’s my real shit. That’s what I’m about.”
+I pushed myself up before the dudes could help me. I believed the shit about family, but [[tonight I could help myself|The Gutter - Outside hub]].]]]
 
-“It’s all about the money these days. Ninjas out there got on 4,000-dollar hatchet man jerseys and they think they the shit.”
+::The Gutter - Outside - Old-school juggalos 6
+“Aw shit,” Large raised his eyebrows, “sorry to hear that. Me and Kyle been bullshitting out here all night. Haven’t seen anything like that.”
 
-“Had a hole in my shirt the other day and I walk by one of them fancy-ass ninjas. You know what he said?”
+Kyle piped up. “Me and Ben’ll keep an eye out though for sure!”
 
-“Naw, man. No way! It ain’t about the clothes!”
+I thanked them and walked back towards the front of [[The Gutter|The Gutter - Outside hub]].
 
-“He called me a juggahoe. You heard that before? I laughed in his face. I told him it was all family, but I’d beat his ass if that was how he needed to be educated.”
+::The Gutter - Golf cart
+{(if: $cartPartsSeen is 4)[I figured I'd done enough damage to the cart, so I decided to keep looking around. But man, [[fuck Ron’s shitty cart|The Gutter - Outside hub]].]}
+\(if: $cartPartsSeen is <4)[Ron drove everywhere in this beaten-to-hell golf cart. I had no idea how he'd gotten it in the first place, especially since he couldn't hold down a job longer than six months. But, he loved to roll around the tarps showing it off every time he added another dumbass accessory to the thing.
```

Comment:
Minor thing, but let's remove the comma after the "But,"

Reply:
Updated!